### PR TITLE
*: add docs for DM Portal

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Add docs and images for DM Portal.
Update TOC.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v4.0 changes for now)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/1679, https://github.com/pingcap/docs-cn/pull/2480
- Other reference link(s):

**Note**: some of the contents in #1679 is outdated. Please cross check [DM Portal](https://pingcap.com/docs-cn/stable/reference/tools/data-migration/dm-portal/) if you need to review this PR.

### CODE REVIEW

File: reference/tools/data-migration/dm-portal.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
+---
```
Comment (lilin90) [inline reference/tools/data-migration/dm-portal.md]: Several images in this file have Chinese characters like 取消, 确定. Please check and update. @WangXiangUSTC @ran-huang

Comment (WangXiangUSTC) [reply reference/tools/data-migration/dm-portal.md]: ok, I will update the images

Comment (ran-huang) [reply reference/tools/data-migration/dm-portal.md]: Screenshots updated. @lilin90 PTAL, thanks~

**Comment by WangXiangUSTC** [comment]
LGTM

File: reference/tools/data-migration/dm-portal.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+- If you run DM Portal locally, visit `127.0.0.1:8280` in your browser.
+- If you run DM Portal on a server, you need to configure a proxy on the server.
+
+### Deploy using DM Ansible
+
+To deploy DM Portal using DM Ansible, refer to [Deploy Data Migration Using DM-Ansible](/how-to/deploy/data-migration-with-ansible.md) for details.
+
+## Usage
+
```
Comment (lilin90) [inline reference/tools/data-migration/dm-portal.md]:
```suggestion

This section describes how to use DM Portal.

```

File: reference/tools/data-migration/dm-portal.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+### Create rules
+
+This feature is used to create a `task.yaml` file. You need to select the replication mode, configure the upstream and downstream instances, configure table routing, and configure the binlog filter.
+
+#### Operation steps
+
+Access the dm-portal page, and click **Create New Rule**.
```
Comment (lilin90) [inline reference/tools/data-migration/dm-portal.md]: Please check the operation order of `You need to select the replication mode, configure the upstream and downstream instances, configure table routing, and configure the binlog filter.` and **Create New Rule** to confirm which one comes first.

File: reference/tools/data-migration/dm-portal.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+![DM Portal BinlogFilter 1](/media/dm-portal-binlogfilter-1.png)
+
+![DM Portal BinlogFilter 2](/media/dm-portal-binlogfilter-2.png)
+
+### Configure table routing
+
+This feature is used to perform the following operations:
+
+- Select the databases and tables that need to be synced, modify their names, and merge databases and tables
```
Comment (lilin90) [inline reference/tools/data-migration/dm-portal.md]: Please confirm whether it's `schemas` or `databases` here. @WangXiangUSTC PTAL

File: reference/tools/data-migration/dm-portal.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+
+    ![DM Portal TableRoute 1](/media/dm-portal-tableroute-1.png)
+
+    ![DM Portal TableRoute 2](/media/dm-portal-tableroute-2.png)
+
+3. Right click the databases and tables to rename them.
+
+    ![DM Portal ChangeTableName](/media/dm-portal-changetablename.png)
+
+4. Select the required table to perform the following operation:
```
Comment (lilin90) [inline reference/tools/data-migration/dm-portal.md]:
```suggestion
4. Select the required table to perform the following operation:

```

File: reference/tools/data-migration/dm-portal.md (hunk 3 above)
Comment (ran-huang) [reply reference/tools/data-migration/dm-portal.md]: Hi, I confirmed with @WangXiangUSTC that the Chinese document should be modified. I have updated this PR and will create a PR to update the Chinese doc later.

File: reference/tools/data-migration/dm-portal.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+
+5. Click **Go Back** to undo the last operation.
+
+    ![DM Portal Revert](/media/dm-portal-revert.png)
+
+6. Click **Reset** to clear the downstream instance.
+
+    ![DM Portal Reset](/media/dm-portal-reset.png)
+
+7. Click **Finish & Download**. DM Portal automatically downloads the `task.yaml` file to the local computer, and creates a `task.yaml` configuration file in the `/tmp/` directory on the DM Portal server.
```
[the remark this answers is not in the document]
Comment (ran-huang) [reply reference/tools/data-migration/dm-portal.md]: Sorry, I added the image.

File: reference/tools/data-migration/dm-portal.md (hunk 4 above)
Comment (ran-huang) [reply reference/tools/data-migration/dm-portal.md]: @lilin90 I suggest using `databases` to make the wording consistent with the images.

File: reference/tools/data-migration/dm-portal.md (hunk 3 above)
Comment (ran-huang) [reply reference/tools/data-migration/dm-portal.md]: @lilin90 PTAL https://github.com/pingcap/docs-cn/pull/2551

File: reference/tools/data-migration/dm-portal.md (hunk 4 above)
Comment (lilin90) [reply reference/tools/data-migration/dm-portal.md]: Agree.

Comment (ran-huang) [reply reference/tools/data-migration/dm-portal.md]: OK. I'll review this document later to make sure the wording consistent 😃

**Pull request merged**